CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift

```diff
@@ -86,6 +86,7 @@ import UIKit
     /**
      Sets the font used in the GiniCapture library by default.
      */
+    @available(*, deprecated, message: "Please use `updateFont(font: UIFont, for textStyle: UIFont.TextStyle)` instead")
```

Comment:
What do you think about adding a todo here to remove this once it's not used anymore (after all screens were updated to the new UI)?

Reply:
Ok, I'll add it.